hw/bsp/apollo3_evb/syscfg.yml

```diff
@@ -48,32 +48,6 @@ syscfg.defs:
         description: 'Pin for LED4'
         value: 17
         
-    UART_0_PIN_TX:
-        description: 'TX pin for UART0'
-        value: 22
-    UART_0_PIN_RX:
-        description: 'RX pin for UART0'
-        value: 23
-    UART_0_PIN_RTS:
-        description: 'RTS pin for UART0'
-        value: 37
-    UART_0_PIN_CTS:
-        description: 'CTS pin for UART0'
-        value: 38
-    
-    UART_1_PIN_TX:
-        description: 'TX pin for UART1'
-        value: 35
-    UART_1_PIN_RX:
-        description: 'RX pin for UART1'
-        value: 36
-    UART_1_PIN_RTS:
-        description: 'RTS pin for UART1'
-        value: 44
-    UART_1_PIN_CTS:
-        description: 'CTS pin for UART1'
-        value: 45
-    
     SPI_0_PIN_CS:
```

Comment:
A bunch of the pin definitions defined here can be removed since they have been relocated to `hw/mcu/ambiq/apollo3/syscfg.yml`